save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/prograssbar/ReactCircleBuilder.kt

```diff
@@ -0,0 +1,49 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE", "FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.externals.prograssbar
+
+import react.ChildrenBuilder
+import react.react
+
+/**
+ * @property hexColor
+ */
+enum class Color(val hexColor: String) {
+    GREEN("#00d500"),
+    RED("#ac0000"),
+    ;
+}
```

Comment:
As far as two different variants of initialization are said to exist, shouldn't you add here two of them? Like `rgb(x, y, z)` and `#KKLLMM`? If one is enough, change the kdoc in `ReactCircleProps`